src/test/ui/unsafe/rfc-2585-unsafe_op_in_unsafe_fn.rs

```diff
@@ -1,16 +1,38 @@
 #![feature(unsafe_block_in_unsafe_fn)]
 #![warn(unsafe_op_in_unsafe_fn)]
 #![deny(unused_unsafe)]
+#![deny(safe_packed_borrows)]
 
 unsafe fn unsf() {}
+const PTR: *const () = std::ptr::null();
+static mut VOID: () = ();
+
+#[repr(packed)]
+pub struct Packed {
+    data: &'static u32,
+}
+
+const PACKED: Packed = Packed { data: &0 };
 
 unsafe fn foo() {
     unsf();
     //~^ WARNING call to unsafe function is unsafe and requires unsafe block
+    *PTR;
+    //~^ WARNING dereference of raw pointer is unsafe and requires unsafe block
+    VOID = ();
+    //~^ WARNING use of mutable static is unsafe and requires unsafe block
+    &PACKED.data; // the level for the `safe_packed_borrows` lint is ignored
```

Comment:
Does this man the warning is shown even with `allow(safe_packed_borrows)`?

Reply:
Yes, this is why I left a comment in the code to gather feedback

Reply:
I see. Yeah not sure what is best.

Reply:
I can see two possible solutions:
- we can use the level defined for `unsafe_op_in_unsafe_fn`, but don't emit anything if `safe_packed_borrow` is `allow`ed
- or we can use the lowest level of both lints

Reply:
> or we can use the lowest level of both lints

Intuitively that makes most sense to me.